src/commcare_cloud/ansible/roles/couchdb/tasks/main.yml

```diff
@@ -52,7 +52,10 @@
   when: not couchdb_path.stat.exists and couchdb_compile|success
 
 - name: Add CouchDB user
-  user: name=couchdb createhome=no password=no state=present
+  user:
+    name: couchdb
+    home: '{{ couchdb_dir }}'
+    state: present
```

Comment:
what does this change do?

Reply:
This gives the couchdb user a home directory, which is necessary because our backup upload script relies on `~/.aws/credentials`. We were properly placing the credentials file at `{{ couchdb_dir }}/.aws/credentials`, but it wasn't getting picked up because that wasn't set as the couchdb user's home dir. (It didn't have a home dir.)